packages/docs/src/DisableFocusTrap.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, ReactNode } from 'react';
+import { DocsContainer, DocsContainerProps } from '@storybook/blocks';
+
+type DisableFocusTrapProps = DocsContainerProps & {
+  children: ReactNode;
+  value?: string;
+};
+
+/**
+ * Custom DocsContainer that disables focus trapping for all syn-side-nav and syn-drawer elements.
+ */
+export const DisableFocusTrap: FC<DisableFocusTrapProps> = ({ children, ...props }) => {
+  const ref = React.useRef<HTMLDivElement>(null);
+  React.useEffect(() => {
+    // Needed otherwise the side-nav / drawer can`t be found
+    setTimeout(() => {
+      // Disable focus trapping for all syn-side-nav`s
```

Comment:
Maybe this is because the element is not yet defined when this is called? How about:

```typescript
useEffect(() => {
  const waitForDefined = async () => {
    await customElements.whenDefined('syn-side-nav');
    await customElements.whenDefined('syn-drawer');
    
    // Further focus trapping logic...
  }

  waitForDefined();
}, []);
```